plutus-benchmark/nofib/test/9.6/queens4-bt.eval.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 8028768773
-| mem: 28840930})
\ No newline at end of file
+({cpu: 8028699773
+| mem: 28840630})
```

Comment:
I've looked at all of these and it _seems_ like these all decrease. Unfortunately dealing with this sort of output is very error-prone, so I opened https://input-output.atlassian.net/browse/PLT-9459?atlOrigin=eyJpIjoiNGM1YTdlNWUyNjhhNGE4OGE3MTUzNmI4OTYwMzQ2MzgiLCJwIjoiaiJ9.